VideoDatabase.js

```diff
@@ -17,8 +17,8 @@
                 // key server to the license server without being visible to the website
                 // or to the authorization service.
                 {
-                    "keyId": "1c817fed-0686-45b6-bce2-d6a4eb873588",
-                    "key": "YV5DwvbXdEU+HSwK+LN3DQ=="
+                    "keyId": "6e5a1d26-2757-47d7-8046-eaa5d1d34b5a",
+                    "key": "GX8m9XLIZNIzizrl0RTqnA=="
                 } 
```

Comment:
These were just some random examples and didn't need changing. While the exact values of these don't matter, since some changes are needed anyway, perhaps change back, so there's no confusion they're somehow related to the hardcoded token below (currently values match whats inside the token).